build-tools/installers/sign-content.proj

```diff
@@ -82,15 +82,20 @@ ourself (using an empty signing identity) before passing these files to ESRP.
     />
   </Target>
 
-  <Target Name="_AddClassicInstallers" >
+  <Target Name="_SignVsix" >
+    <ItemGroup>
+      <FilesToSign Include="$(OutDir)\Xamarin.Android.Sdk-*.vsix">
+        <Authenticode>VsixSHA2</Authenticode>
+      </FilesToSign>
+    </ItemGroup>
+  </Target>
+
+  <Target Name="_SignPkg" >
```

Comment:
nit: It seems like we shouldn't prefix a target with `_` if we call it from the command-line. It feels like it is more "public" if we invoke them that way?

Reply:
Makes sense and should make it easier to type locally whenever needed, I removed the leading `_` from the targets in this file.